examples/simple/src/posts/PostEdit.js

```diff
@@ -65,6 +68,49 @@ const PostEdit = props => (
                 <ImageInput multiple source="pictures" accept="image/*">
                     <ImageField source="src" title="title" />
                 </ImageInput>
+                {permissions === 'admin' && (
+                    <ArrayInput source="authors">
+                        <SimpleFormIterator>
+                            <ReferenceInput
+                                label="User"
+                                source="user_id"
+                                reference="users"
+                            >
+                                <AutocompleteInput />
```

Comment:
an `ArrayInput` listing `AutocompleteInput`?? Why didn't you use an `AutocompleteArrayInput`?

Reply:
Because this is not the only field inside this `ArrayInput`. See the `Role` field below. However, I believe this was introduced as a way to manually test the `AutoCompleteInput` by @ThieryMichel and is not used in any automated tests. Maybe we can remove it ?

Reply:
we may remove it in the future, let's keep it for now